libs/ts-rest/express/src/lib/ts-rest-express.ts

```diff
@@ -264,26 +279,32 @@ const requestValidationErrorHandler = (
       // old-style error handling, kept for backwards compatibility
       if (handler === 'default') {
         if (err.pathParams) {
-          return res.status(400).json(err.pathParams);
+          res.status(400).json(err.pathParams);
+          return;
```

Comment:
I was nervous about removing these returns in #784 but looking at this now, the way ts-rest integrates into express, you can't even chain anything -- even if it is purely an express convention. Thanks again for explaining that in Discord 🙏